tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -2211,7 +2200,7 @@ public function testFormSecuredAndDisabledNotAssoc(): void
         $this->Form->text('Model.text', ['disabled']);
         $this->Form->textarea('Model.textarea', ['disabled']);
         $this->Form->password('Model.password', ['disabled']);
-        $this->Form->radio('Model.radio', [1, 2], ['disabled']);
```

Comment:
This can very well be an array of ids, hence ints.

Reply:
I don't see how integers can be translated into attributes. I assumed this was just a lazy test to "disable" an attribute which is an array key.

Reply:
Updated RadioWidget::_renderInput() to accept `int` for `$text`